cvent-technology-radar.csv

```diff
@@ -88,9 +90,13 @@ Kotlin,Adopt,languages-and-frameworks,FALSE,"<p>Kotlin is used when we need nati
 LaunchDarkly,Trial,Platforms,TRUE,"<p>We're currently in the early phase of using LaunchDarkly across all of Cvent as a commercial version of our experiments platform. We expect to begin broader rollout in 2024.</p>"
 Lerna,Hold,Tools,FALSE,"<p>A tool for managing mono-repositories. Cvent has moved towards pnpm as the tool of choice for this since it replaces npm/yarn and lerna in one tool</p>"
 Maven,Adopt,Tools,FALSE,"<p>The primary java build tool for managing dependencies and various build/deploy best practices at Cvent</p>"
+Mermaid.js,Adopt,Tools,FALSE,"<p>A programmatic flowcharting and diagramming tool.</p>"
 Micro Frontends,Assess,Techniques,FALSE,"<p>Micro frontends is a technique that brings some of the advantages of microservices to the frontend. We're experimenting with various methods that allow us to leverage this technique where appropriate. We've also looked into tools like bit.dev as a mechanism for managing this across the enterprise</p>"
 Micronaut,Hold,languages-and-frameworks,FALSE,"<p>We've chosen spring boot as go forward in 2024.</p>"
 Microservices,Adopt,Techniques,FALSE,"<p>Microservices is an architectural pattern where an application is broken up into discrete mini applications that can be built/deployed independentley and share nothing. This allows each component to be well understood, scaled, and modified independently from other teams</p>"
+Middy,Trial,languages-and-frameworks,TRUE,"<p>Middy is a middleware framework for AWS Lambda on Node.js (Javascript and Typescript applications). We use Middy for authentication and authorization middleware, Hogan Secrets integration, normalization, serialization and validations.</p>"
+mob,Adopt,Tools,TRUE,"<p>mob is a command-line tool orchestrating mob programming discipline through git integration.</p>"
+Mob Programming,Adopt,Techniques,TRUE,"<p>Mob programming is an extension of pair programming applied to the whole agile team in a collective setting.</p>"

```

Comment:
Not that I have anything against mob programming, however I'd expect to see it in trial across a variety of sprint teams before pushing to adopt. Do you know if that has happened?

Reply:
We've seen a decent amount of success with mob programming for the Oktagon team. I'm not sure what other teams may have tried this out, but it's very valuable to up-level new engineers and/or junior engineers. It's also been a critical tool that has show success when working with a remote workforce.